File: brefdic/brefdic-server/src/main/java/forzaverita/brefdic/config/CloudConfig.scala
package forzaverita.brefdic.config

import org.springframework.context.annotation.ImportResource
import org.springframework.context.annotation.Profile
import org.springframework.context.annotation.Configuration

@ImportResource(Array("classpath:META-INF/cloud/cloudfoundry-auto-reconfiguration-context.xml"))
@Profile(Array("cloud"))
@Configuration
class CloudConfig {

}

File: daldic-server/src/main/java/forzaverita/daldic/config/CloudConfig.java
package forzaverita.daldic.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.ImportResource;
import org.springframework.context.annotation.Profile;

@ImportResource("classpath:META-INF/cloud/cloudfoundry-auto-reconfiguration-context.xml")
@Profile("cloud")
@Configuration
public class CloudConfig {

}
